server/src-lib/Hasura/RQL/DDL/Schema/Rename.hs

```diff
@@ -433,3 +446,11 @@ updateRel (QualifiedObject sn tn) rn relDef =
               AND table_name = $3
               AND rel_name = $4
                 |] (Q.AltJ relDef, sn , tn, rn) True
+
+getTableConfig :: QualifiedTable -> Q.TxE QErr TableConfig
```

Comment:
I think this should go in `Hasura.RQL.DDL.Schema.Catalog`, since it’s supposed to encapsulate/abstract away the structure of the catalog in SQL (currently it doesn’t always do that, but it would be nice to move in that direction).